pkgs/applications/emulators/craftos-pc/default.nix

```diff
@@ -16,17 +16,17 @@
 }:
 
 let
-  version = "2.8";
+  version = "2.8.1";
   craftos2-lua = fetchFromGitHub {
     owner = "MCJack123";
     repo = "craftos2-lua";
-    rev = "v${version}";
+    rev = "v2.8";
```

Comment:
Any reason we can't use 2.8.1 release?
https://github.com/MCJack123/craftos2-lua/releases/tag/v2.8.1

Reply:
Oops, the developer mentioned there was no v2.8.1 update for an Accelerated (other version of the software) package so I assumed this was the same. Fixed in latest push.